sdk-actors/src/main/java/io/dapr/actors/runtime/ActorRuntime.java

```diff
@@ -249,6 +250,32 @@ public ActorRuntimeConfig getConfig() {
           .flatMap(m -> ((ActorManager)m).invokeMethod(id, actorMethodName, payload));
   }
 
+  /**
+   * Invokes the specified method for the actor, this is mainly used for cross
+   * language invocation.
+   *
+   * @param actorTypeName   Actor type name to invoke the method for.
+   * @param actorId         Actor id for the actor for which method will be invoked.
+   * @param actorMethodName Method name on actor type which will be invoked.
+   * @param payload         RAW payload for the actor method.
+   * @param reentrancyId    Id used to match reentrant requests.
+   * @return Response for the actor method.
+   */
+  public Mono<byte[]> invoke(String actorTypeName,
```

Comment:
Again, another public method that will force us to keep supporting this, while passing context as a param instead of using Mono's context.

Reply:
Hi all, 

This PR is a beutiful one 👍 and is coming with some super nice additions to the actors.

However, it looks like it is a bit stale atm. @halspang @skyao @xiazuojie , any chance that you 
can look into Arthurs comments. 

Thank you and have a nice weekend!

From a Dapr Java-SDK user :-)

/Dan